src/uct/ib/rc/accel/rc_mlx5.inl

```diff
@@ -1597,8 +1597,11 @@ static void UCS_F_ALWAYS_INLINE uct_rc_mlx5_iface_copy_iov_to_dm(
         uct_rc_mlx5_dm_copy_data_t *cache, size_t hdr_len, const uct_iov_t *iov,
         size_t iovcnt, void *dm, uct_ib_log_sge_t *log_sge)
 {
+    typedef uint64_t misaligned_t UCS_V_ALIGNED(1);
```

Comment:
why needed?

Reply:
It seems to be so wrt existing function that copies contiguous buffer to device memory.